test/bricks-net-api.test.js

```diff
@@ -631,4 +769,30 @@ describe('bricks-net-api', function () {
 			}).done(undefined, done);
 		});
 	});
+	
+	describe('`JSON`', function () {
+		it('should serialize into JSON asynchronously', function (done) {
+			var object = { object: { key: "value" }, array: [ 1, 2, 3] };
+			when(
+				api.JSON(object)
+			).then(function (result) {
+				assert.strictEqual(JSON.stringify(object), result);
```

Comment:
Nit: It's a bit better to use plain strings in unit tests, ex. `assert.strictEqual('{"foo":"bar"}', ...`.

Not really important for this particular one ... although, since there is a nonzero chance someone hacking this code later might think it's a good idea to override `JSON.stringify()`, I'd say it's still worth applying to even this very test, since the thread is not entirely hypothetical.

(Same goes for things like HTTP error codes, where in test it's preferred to use the numeric constants, etc.)